src/loaders/gtdb/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,223 @@
+    """
+    Parse and formate result files generated by the GTDB-TK tool.
+    """
+    gtdb_tk_docs = list()
+
+    result_dir = _locate_dir(root_dir, kbase_collection, load_ver, tool='gtdb_tk')
+
+    # Get the list of directories for batches
+    batch_dirs = [d for d in os.listdir(result_dir) if os.path.isdir(os.path.join(result_dir, d))]
+
+    for batch_dir in batch_dirs:
+
+        # retrieve features from summary files
+        summary_files = ['gtdbtk.ar53.summary.tsv', 'gtdbtk.bac120.summary.tsv']
+        for s in summary_files:
+            summary_file = os.path.join(result_dir, str(batch_dir), s)
+
+            if os.path.exists(summary_file):
+                df = _read_tsv_as_df(summary_file, SELECTED_GTDBTK_SUMMARY_FEATURES, genome_id_col='user_genome')
+                docs = df.apply(_gtdbtk_row_to_doc, args=(kbase_collection, load_ver), axis=1).to_list()
+                gtdb_tk_docs.extend(docs)
+
+    return gtdb_tk_docs
+
+
+def _checkm2_row_to_doc(row, kbase_collection, load_version):
+    # Transforms row from checkm2 quality_report.tsv into ArangoDB collection document
+
+    # parse genome id from the "Name" field.
+    # the "Name" field of checkM2 quality_report.tsv is the base name of input file
+    # (e.g. GCA_013331355.1_ASM1333135v1_genomic, GCF_000970085.1_ASM97008v1_genomic).
+    genome_id = '_'.join(row.Name.split('_')[:2])
+
+    doc = _create_doc(row, kbase_collection, load_version, genome_id, SELECTED_CHECKM2_FEATURES,
+                      pre_fix='checkm2')
+
+    return doc
+
+
+def checkm2(root_dir, kbase_collection, load_ver):
+    """
+    Parse and formate result files generated by the CheckM2 tool.
+    """
+    checkm2_docs = list()
+
+    result_dir = _locate_dir(root_dir, kbase_collection, load_ver, tool='checkm2')
+
+    # Get the list of directories for batches
+    batch_dirs = [d for d in os.listdir(result_dir) if os.path.isdir(os.path.join(result_dir, d))]
+
+    for batch_dir in batch_dirs:
+        # retrieve checkM2 result from 'quality_report.tsv'
+        quality_file = os.path.join(result_dir, str(batch_dir), 'quality_report.tsv')
+
+        if os.path.exists(quality_file):
+            df = _read_tsv_as_df(quality_file, SELECTED_CHECKM2_FEATURES, genome_id_col='Name')
+            docs = df.apply(_checkm2_row_to_doc, args=(kbase_collection, load_ver), axis=1).to_list()
+
+            checkm2_docs.extend(docs)
+
+    return checkm2_docs
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='PROTOTYPE - Generate a JSON file for importing and upserting into ArangoDB by parsing computed '
+                    'genome attributes.')
+    required = parser.add_argument_group('required named arguments')
+    optional = parser.add_argument_group('optional arguments')
+
+    # Required flag arguments
+    required.add_argument('--tools', required=True, type=str, nargs='+',
+                          help='Extract results from tools. (e.g. gtdb_tk, checkm2, etc.)')
```

Comment:
I think we discussed this before and there was a reason not to do this but it'd be nice if this defaulted to running everything

Reply:
`running everything` meaning?

Reply:
👍 default tools to all available tools.

Reply:
:+1: